platform-tests/src/test/java/org/junit/platform/commons/util/ClassNameFilterUtilTests.java

```diff
@@ -41,103 +41,119 @@
 @TestInstance(Lifecycle.PER_CLASS)
 class ClassNameFilterUtilTests {
 
+	//@formatter:off
```

Comment:
Sorry, this is probably a matter of taste but I'd rather see you reverting this formatting change.

Reply:
Revert to keeping the formatter off to methods? Or remove from tests altogether?

I'm perfectly fine with either

Reply:
Yes, please keep it's scope smaller. It should not extend beyond a method body.

Reply:
Makes sense... Will update